lib/prefixes.js

```diff
@@ -51,7 +51,6 @@ Value.hack(require('./hacks/intrinsic'));
 Value.hack(require('./hacks/pixelated'));
 Value.hack(require('./hacks/image-set'));
 Value.hack(require('./hacks/cross-fade'));
-Value.hack(require('./hacks/flex-values'));
```

Comment:
Are you sure about this changes? Looks dangerous.

Reply:
Yeap, it is dangerous. We use it for `transition: flex-grow`.

Reply:
Oh I see... I have returned file.
Would be good to add test for this case.